src/js/main.js

```diff
@@ -0,0 +1,201 @@
+const formBlock = document.getElementById('input-bg');
+
+const modalFilter = document.createElement('div');
+modalFilter.setAttribute('id', 'modal-filter');
+modalFilter.classList.add('check--form');
+formBlock.append(modalFilter);
+
+const modalWindow = document.getElementById('showModal');
+let selected = {
+  children: [],
+};
+const counts = {
+  adult: {
+    count: 0,
+  },
+  room: {
+    count: 0,
+  },
+  child: {
+    count: 0,
+  },
+};
+const elements = {
+    adults: {
+      identifier: 'adults',
+      label: 'Adults',
+      value: 0,
+      count: 0,
+      values: {
+        min: 0,
+        max: 30,
+      },
+    },
+    rooms: {
+      identifier: 'rooms',
+      label: 'Rooms',
+      value: 0,
+      count: 0,
+      values: {
+        min: 0,
+        max: 30,
+      },
+    },
+    children: {
+      identifier: 'children',
+      label: 'Children',
+      value: 0,
+      count: 0,
+      values: {
```

Comment:
`values` looks similar to `value` but their purposes are completely different
Try to rename it to something like `validationRules` and maybe make this configuration dynamic. For example, you can only specify `min` or `max` or some other rules